back/app/policies/user_policy.rb

```diff
@@ -122,10 +122,7 @@ def allowed_custom_field_keys
   end
 
   def enabled_custom_fields
```

Comment:
I think we should find a name without `enabled` in it.

Reply:
@sebastienhoorens I've changed the name to allowed_custom_fields. If you could approve that would be super!